CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 
 ## Prerelease
 
+### Removals
```

Comment:
Nit: Using `Removals` feels like it should be a breaking change. Can this be an update?
```
Updates the `Form` component to remove the `aria-label`. 
The previous value of `"form"` was redundant and not 
required.
```

Reply:
True, but we've been making use of this. I'll update this.